### PR TITLE
Fixing issue 2205

### PR DESCRIPTION
**Pull request by VadimDor**
https://github.com/OpenSCAP/scap-security-guide/issues/2205

### CODE REVIEW

**Comment by mpreisler** [comment]
@openscap-ci test this please

File: shared/templates/static/bash/rpm_verify_permissions.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ declare -a SETPERMS_RPM_LIST
 
 # Create a list of files on the system having permissions different from what
 # is expected by the RPM database
-FILES_WITH_INCORRECT_PERMS=($(rpm -Va --nofiledigest | grep '^.M'))
+FILES_WITH_INCORRECT_PERMS=($(rpm -Va --nofiledigest | grep '^.M' | cut -d ' ' -f4-))
```
Comment (jan-cerny) [inline shared/templates/static/bash/rpm_verify_permissions.sh]: on my Fedora there is one space still left, it wants `-f5-`

Comment (shawndwells) [inline shared/templates/static/bash/rpm_verify_permissions.sh]:
`````
$ sudo rpm -Va --nofiledigest | grep '^.M'
.M.......    /var/lib/docker
$ sudo rpm -Va --nofiledigest | grep '^.M' | cut -d ' ' -f4-
 /var/lib/docker
`````

Comment (shawndwells) [inline shared/templates/static/bash/rpm_verify_permissions.sh]:
`````$ sudo rpm -Va --nofiledigest | grep '^.M' | cut -d ' ' -f5-
/var/lib/docker
`````

(note the extra space before /var/lib/docker is now gone)

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
Did the small fix, merging.
Thanks for you contribution.

**Pull request merged**